clientv3/integration/network_partition_test.go

```diff
@@ -234,7 +234,7 @@ func testBalancerUnderNetworkPartitionWatch(t *testing.T, isolateLeader bool) {
 	wch := watchCli.Watch(clientv3.WithRequireLeader(context.Background()), "foo", clientv3.WithCreatedNotify())
 	select {
 	case <-wch:
-	case <-time.After(3 * time.Second):
+	case <-time.After(maxLeaderChangeWait):
```

Comment:
this should also be a `requestTimeout` (the one we renamed from leaseRequestTimeout). basically we send a watch request to etcd and wait for the ack from the channel from the etcd server side.